package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "datamegh",
+  "version": "1.0.0-alpha.1",
+  "description": "Datamegh - Data Engineering for the cloud.",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "pyright"
```

You should name it differently. The `test` script is supposed to be used for running actual tests - this just runs type checking.

```suggestion
    "typecheck": "pyright"
```